src/new/wallet.js

```diff
@@ -922,8 +843,20 @@ class HathorWallet extends EventEmitter {
 
   /**
    * Connect to the server and start emitting events.
+   *
+   * @param {Object} optionsParams Options parameters
+   *  {
+   *   'pinCode': pin to decrypt xpriv information. Required if not set in object.
+   *   'password': password to decrypt xpriv information. Required if not set in object.
+   *  }
    **/
-  start() {
+  start(optionsParams = {}) {
+    const options = Object.assign({ pinCode: null, password: null }, optionsParams);
+    const pinCode = options.pinCode || this.pinCode;
+    const password = options.password || this.password;
```

Comment:
Isn't password required here?

Reply:
Actually is not required if starting using xpriv. But I should add a verification `if (!xpriv && !password) => error`

Reply:
Done